packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -699,34 +685,18 @@ export class GarbageCollector implements IGarbageCollector {
 		// This is important because a container may never load during a node's Sweep Grace Period,
 		// so that node would directly become sweep-ready skipping over tombstone-ready state,
 		// but should be Tombstoned since Sweep is disabled.
-		const { nodesToTombstone, nodesToDelete } = {
-			nodesToTombstone: [...tombstoneReadyNodes],
-			nodesToDelete: [] as string[],
-		};
-		switch (this.configs.shouldRunSweep) {
-			case "YES":
-				nodesToDelete.push(...sweepReadyNodes);
-				break;
-			case "ONLY_BLOBS":
-				sweepReadyNodes.forEach((nodeId) => {
-					const nodeType = this.runtime.getNodeType(nodeId);
-					if (nodeType === GCNodeType.Blob) {
-						nodesToDelete.push(nodeId);
-					} else {
-						nodesToTombstone.push(nodeId);
-					}
-				});
-				break;
-			default: // case "NO":
-				nodesToTombstone.push(...sweepReadyNodes);
-				break;
-		}
+		const { nodesToTombstone, nodesToDelete } = this.configs.sweepEnabled
```

Comment:
@markfields Since tombstone is part of two-staged sweep, I believe what we can do here is not do anything if sweep is disabled. If sweep is enabled, then the nodes are either tombstone-ready or sweep-ready. It simplifies this code but more than that, it's inline with two-staged sweep where tombstone is not separately controlled.

Reply:
Not sure your point here.  The code in latest commit is functionally equivalent to before (excluding the ONLY_BLOBS case), just written differently.